apps/namadillo/src/App/Transactions/TransactionCard.tsx

```diff
@@ -136,6 +132,14 @@ export const TransactionCard = ({
   const validators = useAtomValue(allValidatorsAtom);
   const validator = validators?.data?.find((v) => v.address === receiver);
 
+  const getBaseAmount = (): BigNumber | undefined => {
+    if (asset && txnInfo?.amount) {
+      if (isBondingTransaction) return toDisplayAmount(asset, txnInfo.amount);
+      if (isNamadaAsset(asset)) return txnInfo.amount;
```

Comment:
ok, I checked the indexer endpoint and it's incompatible, transparent transfers are saved as base 6 and bond are base 0. I think we could request for the indexer team to update this later and use only a common pattern. At least we have the fix on the front :)